docs/2.0/reference/pipelines/feature-flags.md

```
@@ -0,0 +1,101 @@
+**Default Value**: Disabled
+</li>
+<li>**How to Enable**: Set to `"true"`</li>
+</ul>
+
+#### `PIPELINES_FEATURE_EXPERIMENT_COLOCATED_FILE_UNIT_CHANGE_DETECTION`
+<ul>
+<li>
+Enables changes to files within a Terragrunt Unit to be detected as a ModuleChanged job instead of a FileChanged job. A Terragrunt Unit is any directory excluding the root of the repository containing a `terragrunt.hcl` file.
+
+E.g. given a repository containing the following files:
+```
```

markdownlint-cli2 (0.17.2)

Fenced code blocks should have a language specified
null

(MD040, fenced-code-language)